Build failure on Alpine Linux

Hello, I'm trying to package signal-estimator for Alpine Linux, I was originally going to just do a cli only package as I would probably not use the GUI, but I thought I might try to build it anyway and have a -gui subpackage. I am running into the following build error:

```
/srv/aports/repo/signal-estimator/src/signal-estimator-0.0.4/src/gui/MainWindow.cpp: In constructor 'MainWindow::MainWindow(QWidget*)':
/srv/aports/repo/signal-estimator/src/signal-estimator-0.0.4/src/gui/MainWindow.cpp:43:36: error: cannot convert 'QColor' to 'const QPen&'
   43 |     zoomInPicker->setRubberBandPen(QColor(Qt::red));
      |                                    ^~~~~~~~~~~~~~~
      |                                    |
      |                                    QColor
In file included from /usr/include/qwt/qwt_plot_picker.h:14,
                 from /srv/aports/repo/signal-estimator/src/signal-estimator-0.0.4/src/gui/MainWindow.hpp:18,
                 from /srv/aports/repo/signal-estimator/src/signal-estimator-0.0.4/src/gui/MainWindow.cpp:4:
/usr/include/qwt/qwt_picker.h:209:28: note:   initializing argument 1 of 'void QwtPicker::setRubberBandPen(const QPen&)'
  209 |     void setRubberBandPen( const QPen& );
      |                            ^~~~~~~~~~~
```

--- a/src/gui/MainWindow.cpp
+++ b/src/gui/MainWindow.cpp
@@ -6,6 +6,7 @@
 #include "ui_MainWindow.h"
 
 #include <QCheckBox>
+#include <QPen>
 
 #include <qwt_legend.h>
 #include <qwt_picker_machine.h>
@@ -40,7 +41,7 @@ MainWindow::MainWindow(QWidget* parent)
     QwtPlotPicker* zoomInPicker = new QwtPlotPicker(ui->OutputSig->canvas());
     zoomInPicker->setStateMachine(new QwtPickerDragRectMachine());
     zoomInPicker->setRubberBand(QwtPicker::RectRubberBand);
-    zoomInPicker->setRubberBandPen(QColor(Qt::red));
+    zoomInPicker->setRubberBandPen(QPen(QColor(Qt::red)));
     connect(zoomInPicker, qOverload<const QRectF&>(&QwtPlotPicker::selected),
         [this](const QRectF& rect) {
             ui->OutputSig->setAxisScale(

